lib/timeline/Timeline.js

```diff
@@ -202,6 +202,16 @@ function Timeline (container, items, groups, options) {
 // Extend the functionality from Core
 Timeline.prototype = new Core();
 
+Timeline.prototype._activateOnInitialDrawDone = function() {
+  if (!this.initialDrawDone) {
+    this.initialDrawDone = true;
+    setTimeout(() => {
+      if (this.options.onInitialDrawComplete) {
```

Comment:
Why bother with `setTimeOut` if `onInitialDrawComplete` is not present? Better to move this condition out of `setTimeOut`.